core/server/data/migration/fixtures/fixtures.json

```diff
@@ -289,6 +289,26 @@
                     "name": "Delete subscribers",
                     "action_type": "destroy",
                     "object_type": "subscriber"
+                },
+                {
+                    "name": "Browse invites",
+                    "action_type": "browse",
+                    "object_type": "invite"
+                },
+                {
+                    "name": "Read invites",
+                    "action_type": "read",
+                    "object_type": "invite"
+                },
+                {
+                    "name": "Add invites",
+                    "action_type": "add",
+                    "object_type": "invite"
+                },
+                {
+                    "name": "Delete invites",
+                    "action_type": "destroy",
+                    "object_type": "invite"
```

Comment:
Should we maybe add the permission for edit, even if it isn't something that can be done yet? Just an idea.